static/scripts/theme-blog.js

```diff
@@ -3,9 +3,20 @@ const stripHtml = require('string-strip-html');
 const ghostTheme = $('#SC_THEME').text();
 
 function fetchContent() {
+	let finalHtml = '';
+
 	$(`.${ghostTheme}-blog .spinner`).show();
 	$(`.${ghostTheme}-blog .placeholder`).hide();
 
+	const changePage = () => {
+		finalHtml = finalHtml.replace('<pre><code>', '');
```

Comment:
Please don't do tag replacement. Take instead childelement of these tags and set it to finalHtml

Reply:
done, please check ;)